### ISSUE TITLE
UI issues and feature proposal

### ISSUE THREAD

**Issue by AnkitaKhurana** [OpenGenus/quark#11]
**This is a(n):**

- [X] New feature proposal
- [ ] Reporting a keyword
- [X] Error
- [X] Proposal to the Search Engine
- [ ] Other

**Details:**

- The last row is not visible in the current UI.(eg. On clicking tag "sort" last code under merge sort is inaccessible)
- No line space after the display of the last element of any topic.(As visible in the image the last row under quicksort is too close to the next merge sort section)
- One additional UI feature can be added to display the results in columns instead of an entire page to reduce scroll time for the user. 

![image](https://user-images.githubusercontent.com/13145896/36060797-6fdecda2-0e76-11e8-9ca1-6328ab7df0c8.png)

**Comment by AnkitaKhurana** [comment]
I would like to work on this issue.

**Comment by AdiChat** [comment]
Sounds good. Go for it 👍

**Comment by AnkitaKhurana** [comment]
@AdiChat 
I have tried implementing the columns feature to the UI.
And fixed the mentioned errors.
Kindly review the updated UI, and suggest improvements on the same.

![image](https://user-images.githubusercontent.com/13145896/36064252-43990c66-0eae-11e8-85b4-ed8d4785779e.png)

**Comment by AdiChat** [comment]
The UI looks good. 👍 

The UI makes a good use of the available space. However, components can look highly disproportionate (in case of `sort`, the first result has 14 sub-results while the second one has only 1 sub-result).

So, we must experiment with other UIs as well. 
In my opinion, you may keep two columns on the page. Each column will display the results as being displayed in the current production design that is sequentially. In this case, we must ensure that the difference in the size of the two columns is minimized. 
So, in simple terms, just move half of the results to the second column from the first column.

**Comment by AnkitaKhurana** [comment]
Thanks for the suggestions. :+1: 
I have implemented the 2 sub columns approach to divide total queries into halves, to reduce long lists.
Also reduced the result sections to two columns in one page as required. 
Kindly review the changes, 

![image](https://user-images.githubusercontent.com/13145896/36087018-d6a61178-0ff5-11e8-894b-4fe44c87a104.png)

********************************************************************************************************************
![image](https://user-images.githubusercontent.com/13145896/36087083-51b59514-0ff6-11e8-99db-c8ab80d7fde1.png)

**Comment by AdiChat** [comment]
In the above design, the problem of a disproportionate component might occur as well. 

I meant a design like this:

![pre](https://user-images.githubusercontent.com/10634210/36107290-924b2c08-103f-11e8-986c-d783d7544e17.png)

Also, note that there is no alignment between the two columns. 👍
